PhysicsTools/SelectorUtils/interface/CutApplicatorBase.h

```diff
@@ -84,7 +86,7 @@ class CutApplicatorBase : public candf::CandidateCut {
   
   virtual CandidateType candidateType() const { return NONE; }
 
-  const std::string& name() const { return _name; }
+  virtual const std::string& name() const { return _name; }
```

Comment:
Yeah it's only for expr evaluator.

-Lindsey

(Sent from my Nexus 6)
On Jul 9, 2015 7:56 PM, "Slava Krutelyov" <notifications@github.com> wrote:

> In PhysicsTools/SelectorUtils/interface/CutApplicatorBase.h
> <https://github.com/cms-sw/cmssw/pull/9659#discussion_r34283368>:
>
> > @@ -84,7 +86,7 @@ class CutApplicatorBase : public candf::CandidateCut {
> >
> >    virtual CandidateType candidateType() const { return NONE; }
> >
> > -  const std::string& name() const { return _name; }
> > +  virtual const std::string& name() const { return _name; }
>
> this is not in 74X
> is it OK?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9659/files#r34283368>.
>